From a63fead7152bd3bc7d3a4d8cbdc217c8ce8c53c3 Mon Sep 17 00:00:00 2001
From: Kleis Auke Wolthuizen <github@kleisauke.nl>
Date: Tue, 23 Aug 2022 10:29:57 +0200
Subject: [PATCH 02/36] Disable NLS

Upstream-Status: Inappropriate [disable feature]
This patch is just for our convenience.

(cherry picked from commit 1f4fa40ad02655bea49322dad24f02c6c50e78c1)
---
 glib/gi18n-lib.h | 15 +++------------
 1 file changed, 3 insertions(+), 12 deletions(-)

diff --git a/glib/gi18n-lib.h b/glib/gi18n-lib.h
index fe9e79dbd..8597f90d1 100644
--- a/glib/gi18n-lib.h
+++ b/glib/gi18n-lib.h
@@ -20,19 +20,10 @@
 #ifndef __G_I18N_LIB_H__
 #define __G_I18N_LIB_H__
 
-#include <glib.h>
-
-#include <libintl.h>
-#include <string.h>
-
-#ifndef GETTEXT_PACKAGE
-#error You must define GETTEXT_PACKAGE before including gi18n-lib.h.  Did you forget to include config.h?
-#endif
-
-#define  _(String) ((char *) g_dgettext (GETTEXT_PACKAGE, String))
-#define Q_(String) g_dpgettext (GETTEXT_PACKAGE, String, 0)
+#define  _(String) (String)
+#define Q_(String) (String)
 #define N_(String) (String)
-#define C_(Context,String) g_dpgettext (GETTEXT_PACKAGE, Context "\004" String, strlen (Context) + 1)
+#define C_(Context,String) (String)
 #define NC_(Context, String) (String)
 
 #endif  /* __G_I18N_LIB_H__ */
